docs/website/tds/tutorial/Subset.html

```diff
@@ -253,8 +253,8 @@ <h2><a name="CDMRemote">CDMRemote</a></h2>
 
 <p>
 <li><strong>Reference</strong>:
-<a href="https://www.unidata.ucar.edu/software/netcdf-java/v4.6/reference/stream/CdmRemote.html target="_blank">
-https://www.unidata.ucar.edu/software/netcdf-java/v4.6/reference/stream/CdmRemote.html</a>
+<a href="https://www.unidata.ucar.edu/netcdf-java/4.6/userguide/reference/stream/CdmRemote.html target="_blank">
```

Comment:
Spoke with Doug about this.  Let's use the convention of https://www.unidata.ucar.edu/**software**/<package_name>/*

Reply:
> https://www.unidata.ucar.edu/netcdf-java/4.6/userguide/reference/stream/CdmRemote.html

Oops, this should actually be https://docs.unidata.ucar.edu/netcdf-java/4.6/userguide/reference/stream/CdmRemote.html, or better yet `/../../reference/stream/CdmRemote.html`